include/libtorrent/address.hpp

```diff
@@ -73,6 +73,29 @@ namespace libtorrent {
 	using boost::asio::ip::make_address_v4;
 	using boost::asio::ip::make_address_v6;
 #endif
+
+	TORRENT_EXTRA_EXPORT bool is_local(address const& a);
```

Comment:
what do you think about putting these inside `aux_/address.hpp` instead?

Reply:
yeah, that probably makes sense. If I'm adding a new header, I might as well call it something else too.